argparse_dataclass.py

```diff
@@ -485,6 +496,24 @@ def _get_arg_name(field: Field):
     return field.name.replace("_", "-")
 
 
+def _fields(dataclass) -> Tuple[Field]:
+    # dataclass.fields does not return fields that are of type InitVar
+    dc_fields = getattr(dataclass, "__dataclass_fields__", None)
+    if dc_fields is None:
+        return fields(dataclass)
```

Comment:
Using `__dataclass_fields__` seemed to be the only way to get the `InitVar` fields. 
I was not sure what to do if it was not found. I decided to fall back to `dataclasses.fields` because that currently raise an exception if that attribute is not found.